content/en/docs/installation/guide/install-on-gke-operator.md

```diff
@@ -8,30 +8,25 @@ description: >
   Use the Armory Operator to deploy Armory Enterprise for Spinnaker in your Google Kubernetes Engine (GKE) cluster.
 ---
 
-## Overview of installing Armory Enterprise for Spinnaker
+## Overview of installing Armory Enterprise in GKE
 
-This guide contains instructions for installing Armory on a Google Kubernetes Engine (GKE) cluster using the [Armory Operator]({{< ref "operator" >}}). Refer to the [Armory Operator Reference]({{< ref "operator-config" >}}) for manifest entry details.
-
-> If you want to install Spinnaker<sup>TM</sup>, use the open source Operator, which you can download from its GitHub [repo](https://github.com/armory/spinnaker-operator).
-
-## Prerequisites for installing Armory Enterprise and the Armory Operator
-
-* You have a machine configured to use the `gcloud` CLI tool and a recent
-  version of the `kubectl` tool
-* You have logged into the `gcloud` CLI and have permissions to create GKE
-  clusters and a service account
+Installing Armory using the Armory Operator consists of the following steps:
 
-## Armory Enterprise installation summary
+* [Create a cluster](#create-a-gke-cluster) for the Armory Operator and Armory Enterprise
+* [Create a GCS service account](#create-a-gcs-service-account)
+* [Create a Kubernetes service account](#create-a-kubernetes-service-account)
+* [Create a Google Cloud Storage (GCS) bucket](#create-a-gcs-bucket)
+* [Install the Armory Operator](#install-the-armory-operator)
+* [Configure your Armory Enterprise installation](#configure-your-armory-enterprise-installation)
+* [Deploy Armory Enterprise using the Armory Operator](#deploy-armory-enterprise)
 
-Installing Armory using the Armory Operator consists of the following steps:
+## {{% heading "prereq" %}}
```

Comment:
Should we add `prereq` to the vocab list so it stops complaining?

Reply:
I added it to this PR early one but of course it's not effective here...